src/tableDrawer.ts

```diff
@@ -392,9 +392,12 @@ function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
       doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
     }
   } else if (typeof cellStyles.lineWidth === 'object') {
-    doc.rect(cell.x, cursor.y, cell.width, cell.height, 'F')
 
     const sides = Object.keys(cellStyles.lineWidth)
+    if(sides.length === 4) {
+      const fillColor = cellStyles.fillColor === false ? null : typeof cellStyles.fillColor !== "string" ? 'F' : cellStyles.fillColor;
+      doc.rect(cell.x, cursor.y, cell.width, cell.height, fillColor);
```

Comment:
I'm not sure when this line is useful. Maybe need to check the git history to see why it was added.